.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,36 +1,18 @@
-# Required for all PRs
-
-<!-- Before opening a pull request you should run the following checks locally to make sure the CI will pass.
-
-make lint
-make check
-make check-deps
-make test
-make docs
-
+## Summary
+<!-- Mandatory
+Explain here the why, the rationale and motivation, for the changes.
 -->
 
-<!-- Complete the tasks in the following list. Change [ ] to [x] to
-show completion. -->
-
-- [ ] Updated associated README.md.
-- [ ] Wrote appropriate unit tests.
-- [ ] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)
```

Comment:
Should we also keep those as a kind of checklist?

Reply:
Fine with me. Just thought it might make people think if they really did the unit test etc, but as most ppl delete the items anyway it's probably not worth it.